gtsfm/loader/astronet_loader.py

```diff
@@ -55,11 +56,14 @@ def __init__(
             max_frame_lookahead (optional): maximum number of consecutive frames to consider for
                 matching/co-visibility. Any value of max_frame_lookahead less than the size of
                 the dataset assumes data is sequentially captured.
+            max_resolution: integer representing maximum length of image's short side
+               e.g. for 1080p (1920 x 1080), max_resolution would be 1080
```

Comment:
same comment about rephrasing the docstring

Reply:
thanks, updated it